modules/v2x/proto/BUILD

```diff
@@ -129,3 +129,32 @@ py_proto_library(
         "//modules/perception/proto:perception_obstacle_py_pb2",
     ],
 )
+
+cc_proto_library(
```

Comment:
Generate this BUILD file by `./scripts/proto_build_generator.py modules/v2x/proto/BUILD` from apollo root dir.